freeclimb-cs-sdk/api/call/CallsSearchFilters.cs

```diff
@@ -146,6 +149,18 @@ public virtual string toJson()
         /// <param name="val">parentCallId value.</param>
         public void setParentCallId(string val) { this.parentCallId = val; }
 
+        /// <summary>
+        /// Retrieve the active value.
+        /// </summary>
+        /// <returns>The active value of the object.</returns>
+        public bool getActive { get { return this.active; } }
+
+        /// <summary>
+        /// Sets the active object value.
+        /// </summary>
+        /// <param name="val">The active value.</param>
+        public void setActive(bool val) { this.active = val; }
```

Comment:
Not sure if this is needed unless it's set and unset every time the call status changes.